segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeperstorage/BookKeeperStorageConfig.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.bookkeeperstorage;
+
+import io.pravega.common.util.ConfigBuilder;
+import io.pravega.common.util.ConfigurationException;
+import io.pravega.common.util.InvalidPropertyValueException;
+import io.pravega.common.util.Property;
+import io.pravega.common.util.TypedProperties;
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.util.Arrays;
+import lombok.Getter;
+
+public class BookKeeperStorageConfig {
```

Comment:
Do we want to have a separate config for Tier 1 and 2 when using the log storage?

Reply:
This gives users choice to use separate BKs if they wants to. If they want to use the same BK, the config can point to the same values.